training/general/src/environments/sklearn-1.1/tests/src/main.py

```diff
@@ -13,17 +12,12 @@
 
 # define functions
 def main(args):
-    """Run and evaluate model."""
-    # enable auto logging
-    mlflow.autolog()
```

Comment:
was the mlflow.autolog() removed on purpose? why?

Reply:
@rtanase: we removed mlflow for some testing we were doing with sklearnex in the container, you can keep the line and mlflow related calls in the code - no concern. I reverted the changes to have mlflow included just as before